### PR TITLE
Setup travis to run eslint

### PR DESCRIPTION
**Pull request by mauriciovieira**
Run eslint for the first 3 directories.

### CODE REVIEW

File: api_keys/collection/schema.js [hunk 1]
```diff
@@ -1,7 +1,9 @@
+import { SimpleSchema } from 'meteor/aldeed:simple-schema';
+
 import { ApiKeys } from './';
```
Comment (marla-singer) [inline api_keys/collection/schema.js]: @mauriciovieira When `export default` is used, don't use brackets. Must be
```javascript
import ApiKeys from './';
```

File: .eslintrc [hunk 2]
```diff
@@ -23,6 +23,10 @@
       2,
       "always-multiline"
     ],
+    "func-names": [
```
Comment (mauriciovieira) [inline .eslintrc]: @brylie I already added.

File: api_catalog/client/api_catalog.js [hunk 3]
```diff
@@ -112,7 +117,7 @@ Template.apiCatalog.onCreated(function () {
   });
 });
 
-Template.apiCatalog.onRendered(function () {
+Template.apiCatalog.onRendered(() => {
```
Comment (brylie) [inline api_catalog/client/api_catalog.js]: In Template lifecycle callbacks, we need to use regular function definitions. Regular functions preserve the lexical scoping of `this`, which refers to the Template instance. When using fat arrow functions in Template lifecycle callbacks, we lose the reference to the Template instance.

File: api_keys/server/methods.js [hunk 4]
```diff
@@ -1,11 +1,17 @@
...
 import { TAPi18n } from 'meteor/tap:i18n';
+
 import { ApiKeys } from '/api_keys/collection';
-import { ProxyBackends } from '/proxy_backends/collection';
 import { Proxies } from '/proxies/collection';
+import { ProxyBackends } from '/proxy_backends/collection';
+
 
 Meteor.methods({
   createApiKey (idApi) {
```
Comment (brylie) [inline api_keys/server/methods.js]: Change argument to be `apiId`.

File: api_keys/server/methods.js [hunk 5]
```diff
@@ -1,11 +1,17 @@
...
+    // Make sure idApi is a string
+    check(idApi, String);
```
Comment (brylie) [inline api_keys/server/methods.js]: Change argument to be `apiId`.

File: api_catalog/client/api_catalog.js (hunk 3 above)
Comment (brylie) [inline api_catalog/client/api_catalog.js]: We will need to either ignore the linter rule for this line, or perhaps figure out how to not require arrow functions for Template lifecycle callbacks.

File: .eslintrc (hunk 2 above)
Comment (brylie) [inline .eslintrc]: Cool.

**Comment by brylie** [comment]
Requested changes.

File: api_catalog/client/api_catalog.js (hunk 3 above)
Comment (mauriciovieira) [inline api_catalog/client/api_catalog.js]: I'll ignore it by now.

File: api_keys/server/methods.js (hunk 4 above)
Comment (mauriciovieira) [inline api_keys/server/methods.js]: 👍

File: api_keys/server/methods.js (hunk 5 above)
Comment (mauriciovieira) [inline api_keys/server/methods.js]: 👍

File: api_keys/server/methods.js [hunk 6]
```diff
@@ -8,15 +8,15 @@ import { ProxyBackends } from '/proxy_backends/collection';
...
+  createApiKey (apiId) {
+    // Make sure apiId is a string
+    check(apiId, String);
 
     // Get logged in user
     const currentUser = Meteor.user();
     // Check currentUser exists
     if (currentUser) {
-      const proxyBackend = ProxyBackends.findOne({ apiId: idApi });
+      const proxyBackend = ProxyBackends.findOne({ apiId: apiId });
```
Comment (brylie) [inline api_keys/server/methods.js]: Cool beans. Now that the object key and value match, we can express this line more concisely:

```js
 ..... ProxyBackends.findOne({ apiId });
```

**Comment by mauriciovieira** [comment]
We lost the previous checks because I rebased this branch on develop.

**Pull request merged**